query-engine/connectors/sql-query-connector/src/query_builder/write.rs

```diff
@@ -188,6 +200,7 @@ pub fn create_relation_table_records(
         insert.values(values)
     });
 
+    // NOTE: There is no comment support for MultiRowInsert
```

Comment:
Well... shound't there be?

Reply:
Well, the struct has no comment method, that is what I was referring, I will remove the comment to avoid confussion